app/src/main/assets/queries/nearby_query.rq

```diff
@@ -13,7 +13,7 @@ SELECT
      SERVICE wikibase:around {
        ?item wdt:P625 ?location.
        bd:serviceParam wikibase:center "Point(${LONG} ${LAT})"^^geo:wktLiteral.
-       bd:serviceParam wikibase:radius "${RADIUS}" . # Radius in kilometers.
+       bd:serviceParam wikibase:radius "${RAD}" . # Radius in kilometers.
```

Comment:
The nearby list and map work okay for me with these devices.

* Android 6.0.1, API 23 (real device)
* Android 7.1.1, API 25 (emulator)
* Android 5.1.1, API 22 (emulator)

Sorry about the oversight - the previous patch was incomplete.